posix-configs/SITL/init/test/cmd_template.in

```diff
@@ -25,7 +25,7 @@ mavlink start -x -u 14556 -r 2000000
 mavlink boot_complete
 
 @cmd_name@ start
-sleep 1
+sleep 3
```

Comment:
Why is that?

Reply:
To give the work queue test more time to run. It needs to be structured differently, likely having the main block until completion.